### PR TITLE
NOJIRA Update the readme

### PR DESCRIPTION
**Pull request by sem-hmcts**
### JIRA link (if applicable) ###
N/A

### Change description ###
Update the readme to reflect the actual steps taken during onboarding.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,83 +17,87 @@
...
-
-Creating and starting the containers:
+  a. Create docker network
+  ```bash
+  ./ccd init
+  ```
+  
+  b. Export environment variables
+
+  CCD Data Store API and CCD Definition Store API require a set of environment variables which can be set up by executing the following script.
```
Comment (smathangi) [inline README.md:77]: Can we just say CDM apps (as they include ccd, mca, cpo as well)

File: README.md [hunk 2]
```diff
@@ -152,8 +168,27 @@ At this point most users can run the following 4 scripts
 ./bin/add-ccd-roles.sh
 ```
 
-to get their IDAM environment ready and then move on to the [Ready for take-off](###Ready-for-take-off) section.
 
+Create IDAM users and roles
```
Comment (smathangi) [inline README.md:172]: i think it's CCD roles rathe than `IDAM users and roles`

File: README.md (hunk 1 above)
Comment (sem-hmcts) [reply README.md]: amended

File: README.md (hunk 2 above)
Comment (sem-hmcts) [reply README.md]: corrected

**Review by smathangi — APPROVED** [review]

**Pull request merged**